cicecore/cicedyn/general/ice_init.F90

```diff
@@ -2311,8 +2311,10 @@ subroutine input_data
          write(nu_diag,1033) ' histfreq         = ', histfreq(:)
          write(nu_diag,1023) ' histfreq_n       = ', histfreq_n(:)
          write(nu_diag,1031) ' histfreq_base    = ', trim(histfreq_base)
-         write(nu_diag,1011) ' hist_avg         = ', hist_avg
-         if (.not. hist_avg) write(nu_diag,1039) ' History data will be snapshots'
+         do n=1,max_nstrm
+            write(nu_diag,1011) ' hist_avg         = ', hist_avg(n)
+            if (.not. hist_avg(n)) write(nu_diag,1022) ' History snapshots: ',n
```

Comment:
histfreq and histfreq_n just above this are written with (:) instead of a loop.  Would that make sense here too?  I'm thinking that a vertical list of true/false isn't going to provide very useful information, just take up space.